core/modules/layout/layout.admin.inc

```diff
@@ -65,8 +67,13 @@ function layout_list_page() {
         $storage = t('Custom');
       }
 
+      // Create a link to the settings page.
+      $info = layout_get_layout_template_info($layout->layout_template);
```

Comment:
Instead of repeating this call to the function for each specific layout, can we please move this in a `$all_layout_templates` variable (or similar name), and move it outside the `foreach`? We can then use the array to grab the info for each individual layout/template.

Reply:
Ooh, nice! Good call 😄